### PR TITLE
fix: lint file names

### PR DESCRIPTION
**Pull request by talamcol**
References [GROW-188](https://autoricardo.atlassian.net/browse/GROW-188)

## Motivation and context

Enforces:
- PascalCase for all the components
- camelCase for all ts/js files
- kebab-case for next pages

Unfortunately we cannot lint kebab-case for the assets because they are not picked up by eslint.

### CODE REVIEW

File: src/react.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports = {
     'plugin:testing-library/react',
     'plugin:cypress/recommended',
   ],
-  plugins: ['react', 'testing-library'],
+  plugins: ['react', 'testing-library', 'unicorn'],
```
Comment (quangpdt) [inline src/react.js:13]: Since the plugin was imported `index.js`, we may don't need to re-declare it here

File: src/react.js [hunk 2]
```diff
@@ -20,4 +20,20 @@ module.exports = {
     'react/react-in-jsx-scope': 0,
     'react/jsx-uses-react': 0,
   },
+  overrides: [
+    {
+      files: ['**/components/**/*.tsx', '**/components/**/*.jsx'],
```
Comment (quangpdt) [inline src/react.js:25]: I think we can just set the wildcard for specified ext `*.tsx` and `*.jsx` and The container directory should not be part of the lint rule. WDYT?

Comment (talamcol) [reply src/react.js:25]: will that work for the nextjs pages? They need to be in kebab-case

Comment (quangpdt) [reply src/react.js:25]: Can we make override rules for the `pages` dir?

Comment (talamcol) [reply src/react.js:25]: ah yeah it works because I overwrite it again for the files in the pages directory in the nextjs file

**Review by quangpdt — APPROVED** [review]
Looks good to me. Thanks for the new rules

**Pull request merged**